package.json

```diff
@@ -59,6 +59,7 @@
     "jest": "^29.5.0",
     "jest-environment-miniflare": "^2.14.0",
     "lint-staged": "^13.2.2",
+    "node-fetch": "^3.3.2",
```

Comment:
Can't we use a higher node version instead of `.node-version` and use node's own fetch instead of a 3rd party library? @TheUnderScorer

Reply:
Well, this is not easy to tell for me, but it looks to be alright.

Cloudflare integration doesn't really care about the `.node-version` file because it runs in the Cloudflare Worker Runtime, only our workflow files do. Workflows use `jest` and also `playwright` to run unit tests, and also run `rollup` to prepare a release.

We use `miniflare` as the test environment, as seen [here](https://github.com/fingerprintjs/fingerprintjs-pro-cloudflare-worker/blob/main/jest.config.js#L4). I don't know how node version affects miniflare, maybe it doesn't affect at all. We can try and see how it goes. The same applies to `playwright` as well.

`rollup` config [file](https://github.com/fingerprintjs/fingerprintjs-pro-cloudflare-worker/blob/main/rollup.config.js) shouldn't really care which node version we use. We can again, try and see.

So, we can change the `.node-version` to 20 and see how it goes 👍

Reply:
Seems OK. Nice 👍